rotkehlchen/chain/ethereum/transactions.py

```diff
@@ -115,7 +117,15 @@ def query(
             f_to_ts = filter_query.to_ts
             from_ts = Timestamp(0) if f_from_ts is None else f_from_ts
             to_ts = ts_now() if f_to_ts is None else f_to_ts
-            for address in accounts:
+            for i, address in enumerate(accounts):
```

Comment:
What is this ???

Reply:
This is to notify the frontend the number of accounts that will be queried for this time range. So you know you are in the i aaccoutn of the `len(accounts)` What I did wrong is to use single letter variable

Reply:
I am sorry but I still do not understand what this is. What purpose does it serve? We don't care about this, do we? Where is this useful?

Reply:
This is passed later to the frontend in the message https://github.com/rotki/rotki/pull/4330/files#diff-9e03fec7cdf8f66f69931ce6e1c42f6dd1c7e23218c45cacbcb4dbeb3424a3acR125

So the frontend knows that it has queried `i+1` addresses of `len(accounts)`(total)